src/Relation/Binary/HeterogeneousEquality.agda

```diff
@@ -234,22 +234,22 @@ module ≅-Reasoning where
 
   infix 4 _IsRelatedTo_
 
-  data _IsRelatedTo_ {A : Set ℓ} {B : Set ℓ} (x : A) (y : B) : Set ℓ where
+  data _IsRelatedTo_ {A : Set a} {B : Set b} (x : A) (y : B) : Set a where
```

Comment:
And... the `Level` of the definition now also needs to be changed: if you write `Set _`, then Agda will correctly infer the (minimum necessary) level as `Set (a ⊔ b)` but better to be explicit here, I think.
```suggestion
  data _IsRelatedTo_ {A : Set a} {B : Set b} (x : A) (y : B) : Set (a ⊔ b) where
```

Reply:
I have verified that just `a` works (it type-checks locally with Agda 2.7.0.1), because the level `b` will actually be equal to `a` in the single constructor `relTo`. Due to the same reason, the equality itself also have a similar level constraint:
```agda
data _≅_ {A : Set a} (x : A) : {B : Set b} → B → Set a where
   refl : x ≅ x
```

Reply:
Good catch! My mistake.